src/components/WideCard.tsx

```diff
@@ -1,22 +1,56 @@
 import React from "react";
-import { ReactComponent as ArrowIcon } from "@assets/arrow.svg";
+import { ButtonLink } from "@components";
 
 interface Props {
   className?: string;
   title?: string;
-  children: React.ReactNode;
+  text?: string;
+  target?: string;
+  caption?: string;
+  linkButtonVariant?: "light" | "dark" | "search";
+  bg?: number;
+  newTab?: boolean;
 }
 
-const WideCard: React.FC<Props> = ({ className, children, title }) => {
+const WideCard: React.FC<Props> = ({
+  className,
+  text,
+  title,
+  target,
+  caption,
+  linkButtonVariant,
+  bg,
+  newTab
+}) => {
+  const bgMapping = [
+    "bg-illustration-4-wide mask-illustration-vertical md:mask-illustration-horizontal bg-no-repeat bg-[position:right_-80px_top_-40px]",
+    "bg-community-messages bg-no-repeat md:bg-[position:right_-20px_top_50px] bg-[position:left_-25px_bottom_-50px] scale-110 md:scale-125 mt-36",
+  ];
+
   return (
-    <div className={`container-width-hero mt-20 h-full ${className || ""}`}>
-      <div className="bg-tolo-green relative overflow-hidden">
-        <div className="px-[4%] py-8 md:p-12 text-white absolute items-start md:w-[60%] h-full flex flex-col justify-between">
-          {title && <h1 className="font-semibold text-5xl font-aktiv">{title}</h1>}
-          {children && <p className="font-medium text-2xl mt-8">{children}</p>}
-          <a href="#" className="bg-black z-10 inline-flex items-center bg-opacity-20 opacity-90 rounded-full py-4 px-5 mt-8">Search for Resources<ArrowIcon className="h-5 w-5 ml-8 text-white"/></a>
+    <div className={"container-width lg:container-width-hero mt-20 h-full"}>
+      <div className="bg-tolo-green relative overflow-hidden z-10">
+        <div className="px-[4%] py-8 md:p-12 text-white absolute items-start md:w-[65%] h-full flex flex-col justify-start gap-6 z-20">
+          {title && (
+            <h1 className="font-semibold text-4xl lg:text-5xl font-aktiv">
```

Comment:
This font size is supposed to be 48px according to Figma. It's not in the tailwind scale, so you need to use arbitrary value (or create a custom class in tailwind config – if this is used in multiple places).